common/utils.py

```diff
@@ -20,12 +20,17 @@
 
 ROOT_DIR = os.path.abspath(os.path.dirname(os.path.dirname(__file__)))
 
+assert not (os.getenv('FORCE_NOT_LOCAL') and os.getenv('FORCE_LOCAL')), (
+    'You can\'t set FORCE_LOCAL and FORCE_NOT_LOCAL environment variables to '
+    'True at the same time. If you haven\'t set either of these and/or don\'t '
+    'understand why this is happening please file a bug.')
+
```

Comment:
In the future consider dropping these flags and have a single flag like 'RUNNING_ON_GCE' or 'GOOGLE_CLOUD_LOGGING'. Local logging should be the default not just for presubmit but in general for any tool or unit test started manually, so developers never have to specify this flag. (with this PR we still need to)

Having to set this flag for the GCE service on the other hand is fine, because it'd be set in scripts anyway.